.github/workflows/codeql-analysis.yml

```diff
@@ -25,6 +25,8 @@ on:
       - "*.*.*-*"
       - "v*.*.*-*"
   pull_request:
+    branches:
+      - master
```

Comment:
This was pointed out to us by CodeQL so I fixed it. However, with this fix, CodeQL will work only for PRs to master. I think we need to discuss if this fix is really a good idea.

Reply:
📝 Oral discussions
This change is OK.
In the case of sub-branches, fix it in the main PR that is merged into master.